src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -691,9 +669,9 @@ private void disposeDebuggingCtor()
         [Conditional("NUI_DISPOSE_DEBUG_ON")]
         private void disposeDebuggingDispose(DisposeTypes type)
         {
-            DebugFileLogging.Instance.WriteLog($"swigCMemOwn:{swigCMemOwn} type:{GetType()} copyNativeHandle:{swigCPtrCopy.Handle.ToString("X8")} HasBody:{HasBody()}");
+            DebugFileLogging.Instance.WriteLog($"swigCMemOwn:{swigCMemOwn} type:{GetType()} copyNativeHandle:{swigCPtr.Handle.ToString("X8")} HasBody:{HasBody()}");
 
-            if (swigCPtr.Handle != IntPtr.Zero && swigCPtrCopy.Handle != IntPtr.Zero && HasBody())
+            if (swigCPtr.Handle != IntPtr.Zero && swigCPtr.Handle != IntPtr.Zero && HasBody())
```

Comment:
`swigCPtr.Handle != IntPtr.Zero` Checked twice.

Reply:
thank you very much.
I have changed following your comment.